core/include/drivers/rtc.h

```diff
@@ -110,29 +110,28 @@ int rtc_timecmp(struct optee_rtc_time *a, struct optee_rtc_time *b);
 
 /**
  * rtc_diff_calendar_ms() - Return the difference in milliseconds between
- * two times captures. Assume second time capture @ref2 is older than first time
- * capture @ref1.
+ * two times captures.
  * @ref1: First time capture
  * @ref2: Second time capture
  *
  * Return the time difference in milliseconds.
```

Comment:
Could you explicitly state that it's the computed diff is `ref1 - ref2`. Ditto at line 129.
Also mention here that `LLONG_MAX` is returned in case of overflow.
E.g: `* Return @ref1 - @ref2 in milliseconds or LLONG_MAX in case of overflow`